src/zarr/testing/store.py

```diff
@@ -149,6 +149,58 @@ async def test_read_only_store_raises(self, open_kwargs: dict[str, Any]) -> None
         ):
             await store.delete("foo")
 
+    async def test_with_read_only_store(self, open_kwargs: dict[str, Any]) -> None:
+        kwargs = {**open_kwargs, "read_only": True}
+        store = await self.store_cls.open(**kwargs)
```

Comment:
```suggestion
        store = await self.store_cls.open(read_only=True, **open_kwargs)
```

Reply:
This leads to `TypeError: zarr.abc.store.Store.open() got multiple values for keyword argument 'read_only'`